### PR TITLE
Review.rst grammer fixed

### PR DESCRIPTION
**Pull request by HackerAbhishek7**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: docs/Developers/Review.rst [hunk 1]
```diff
@@ -142,7 +135,7 @@ For the Reviewers
    provides a guideline for how to test bears.
 -  Make sure that the changes in the PR haven't been pushed from the master
    branch. If this has happened, then drop a comment telling the
-   contributor that it is not a good practice, but also indicate
+   a contributor that it is not a good practice, but also indicate
```
Comment (Samsomyajit) [inline docs/Developers/Review.rst]: There should not be "a" there is "The" in the previous line

File: docs/Developers/Review.rst [hunk 2]
```diff
@@ -48,8 +47,7 @@ member will look over it and perform the merge if everything is good.
 Best Practices for Review Discussion
 ------------------------------------
 1. After the PR has been reviewed, don't comment on a review unless
-   there is a technical discussion going on, or comment only if you
-   are not in agreement with the reviewer. For an addressed status
+   there is a technical discussion going on, or comment only if you are not in agreement with the reviewer. For an addressed status
```
Comment (Chaitanyassr) [inline docs/Developers/Review.rst:50]: use "-" to make it as a point .
example - 
- there is a tech

File: docs/Developers/Review.rst [hunk 3]
```diff
@@ -18,11 +18,10 @@ code of others and share what you have learnt with them. You can use acks and
 unacks like everyone else and ``corobo`` even allows you to set PRs to WIP.
 Check the section below for more information.
 
-Generally follow this process:
+Generally, follow this process:
 
 1. Check if the change is helping us. Sometimes people propose changes that are
-   only helpful for specific usecases but may break others. The concept has to
-   be good. Consider engaging in a discussion on gitter if you are unsure!
+   only helpful for specific use-cases but may break others. The concept has to be good. Consider engaging in a discussion on Gitter if you are unsure!
```
Comment (Chaitanyassr) [inline docs/Developers/Review.rst:24]: not needed.

File: docs/Developers/Review.rst [hunk 4]
```diff
@@ -115,20 +113,15 @@ For the Reviewers
    https://coala.io/review.
 -  Check the commit message.
 -  Read and try to understand the code. If something looks ineffective or
-   bug prone, leave a comment.
-   If in doubt, let the code-writer explain their reasoning until
-   reviewers have understood the code.
--  Generated code is not intended to be reviewed. Instead rather try to
-   verify that the generation was done right. The commit message should
-   expose that.
+   bug-prone, leave a comment.
```
Comment (Chaitanyassr) [inline docs/Developers/Review.rst:116]: add this "-" before staring the line.

File: docs/Developers/Review.rst [hunk 5]
```diff
@@ -115,20 +113,15 @@ For the Reviewers
...
+   If in doubt, let the code-writer explain their reasoning until reviewers have understood the code.
```
Comment (Chaitanyassr) [inline docs/Developers/Review.rst:117]: add this "-" before staring the line.

File: docs/Developers/Review.rst [hunk 6]
```diff
@@ -115,20 +113,15 @@ For the Reviewers
...
+-  Generated code is not intended to be reviewed. Instead, rather try to
+   verify that the generation was done right. The commit message should expose that.
```
Comment (Chaitanyassr) [inline docs/Developers/Review.rst:119]: add this "-" before staring the line.

File: docs/Developers/Review.rst [hunk 7]
```diff
@@ -115,20 +113,15 @@ For the Reviewers
...
 -  Every commit is reviewed independently from the other commits.
 -  Tests should pass for each commit. If you suspect that tests might
-   not pass and a commit is not checked by continuous integration, try
-   running the tests locally.
--  Check the surroundings. In many cases people forget to remove the
-   import when removing the use of something or similar things. It is
-   usually good to take a look at the whole file to see if it's still
-   consistent.
+   not pass and a commit is not checked by continuous integration, try running the tests locally.
```
Comment (Chaitanyassr) [inline docs/Developers/Review.rst:122]: add this "-" before staring the line.

File: docs/Developers/Review.rst [hunk 8]
```diff
@@ -115,20 +113,15 @@ For the Reviewers
...
+-  Check the surroundings. In many cases, people forget to remove the
+   import when removing the use of something or similar things. It is usually good to take a look at the whole file to see if it's still consistent.
```
Comment (Chaitanyassr) [inline docs/Developers/Review.rst:124]: add this "-" before staring the line.

File: docs/Developers/Review.rst [hunk 9]
```diff
@@ -152,8 +145,7 @@ For the Reviewers
     make sure that the patch solves the issue and doesn't create any
     further issues.
 
-    You need to thoroughly review the code, i.e. understand the functionality
-    of the code, check whether it is efficient or not, and leave critical
+    You need to thoroughly review the code, i.e. understand the functionality of the code, check whether it is efficient or not, and leave critical
```
Comment (Chaitanyassr) [inline docs/Developers/Review.rst:148]: not needed.

**Pull request closed without merging**